src/ws-egl.cpp

```diff
@@ -294,7 +294,7 @@ void ImplEGL::foreachDmaBufModifier(std::function<void (int format, uint64_t mod
         uint64_t modifiers[64];
         EGLint numModifiers;
         if (!s_eglQueryDmaBufModifiersEXT(m_egl.display, formats[i], 64, modifiers, NULL, &numModifiers))
-            assert(!"Linux-dmabuf: Failed to query modifiers of a format");
+            continue;
```

Comment:
When this fails, `numModifiers` should be set to 0, allowing `DRM_FORMAT_MOD_INVALID` to be passed through. This way the format doesn't go ignored.

Reply:
Done :+1: